safe/gui/tools/peta_bencana_dialog.py

```diff
@@ -554,3 +579,57 @@ def download(self, url, output_path):
                 self,
                 self.tr('Download error'),
                 self.tr(message))
+
+    def get_available_area(self):
+        """Function to automatically get the available area on API.
+           *still cannot get string data from QByteArray*
+        """
+        available_area = []
+        network_manager = QNetworkAccessManager()
+        api_url = QUrl('https://data.petabencana.id/cities')
+        api_request = QNetworkRequest(api_url)
+        api_response = network_manager.get(api_request)
+        data = api_response.readAll()
+        json_response = QScriptEngine().evaluate(data)
+        geometries = json_response.property('output').property('geometries')
+        iterator = QScriptValueIterator(geometries)
+        while iterator.hasNext():
+            iterator.next()
+            geometry = iterator.value()
+            geometry_code = (
+                geometry.property('properties').property('code').toString())
+            available_area.append(geometry_code)
+
+        display_warning_message_box(
+            self,
+            'output',
+            ', '.join(available_area)
+        )
+
+    def populate_combobox(self):
+        """Populate combobox."""
+        if self.radioButton_prod.isChecked():
```

Comment:
so we need to rename all object that generated from QtDesigner? Or just the one that used in the code?